learning-ojs/en/statistics.md

```diff
@@ -64,6 +64,32 @@ When Geographical and Institutional Statistics have been enabled (see the [Admin
 
 This information is also available via the COUNTER SUSHI interface. See the [Administrator's Guide](https://docs.pkp.sfu.ca/admin-guide/en/statistics#counter) for details.
 
+## Institutional Statistics
```

Comment:
@rcgillis, the guide already touches just briefly on institutional statistics just above (in a 3rd-level heading); the institution stuff isn't just for statistics, but also institutional subscriptions so maybe it's worth renaming this section "Institutions" and discussing both aspects. In any case, since you're altering the TOC, please make similar adjustments to the SUMMARY.md file so that the TOC there is up to date.